webapp/src/ts/modules/tasks/tasks-content.component.ts

```diff
@@ -228,6 +233,19 @@ export class TasksContentComponent implements OnInit, OnDestroy, AfterViewInit {
       });
   }
 
+  private preloadTaskGroupContact(action) {
+    this.tasksActions.setTaskGroupContactLoading(true);
+    return this.tasksForContactService
+      .getLeafTypePlaceParent(action?.content?.contact?._id)
```

Comment:
I went with this name because I wanted to make it clear why this exists upon first read, that we are pre-loading some contact, no matter what it is, into the store, for another component to consume. 

I purposefully didn't use `get` because I expect `get` to return something (which in this case, it doesn't). Maybe `storeTaskGroupContact` ?

Reply:
I think I'm confused
`getLeafTypePlaceParent` returns an object `{doc: contact, type: type}`, right? It's not storing 🤔

Reply:
Yea, but we don't actually return the value, we just store it: 
```
 this.tasksActions.setTaskGroupContactLoading(true);
    return this.tasksForContactService
      .getLeafTypePlaceParent(action?.content?.contact?._id)
      .then(contact => {
        this.tasksActions.setTaskGroupContact(contact);
      })
      .catch(err => {
        console.error('Error when loading task group contact', err);
        this.tasksActions.setTaskGroupContact(null);
      });
```

Reply:
Ah! I think I understand now: 
You're not requesting to change `preloadTaskGroupContact`, but `getLeafTypePlaceParent`?

Reply:
Sorry for the confusion!

Reply:
How about `getLeafPlaceAncestor` ?

Reply:
`getLeafPlaceAncestor` <- I understand this one better 🙂 Thanks!